x-pack/plugins/security_solution/cypress/e2e/exceptions/entry/multiple_conditions.cy.ts

```diff
@@ -0,0 +1,124 @@
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EXCEPTION_ITEM_ACTIONS_BUTTON, REMOVE_EXCEPTION_BTN } from '../../../screens/rule_details';
+import { deleteAlertsAndRules } from '../../../tasks/common';
+import { getNewRule } from '../../../objects/rule';
+
+import { createRule } from '../../../tasks/api_calls/rules';
+import { goToRuleDetails } from '../../../tasks/alerts_detection_rules';
+import { esArchiverResetKibana, esArchiverUnload } from '../../../tasks/es_archiver';
+import { login, visitWithoutDateRange } from '../../../tasks/login';
+import {
+  openExceptionFlyoutFromEmptyViewerPrompt,
+  goToExceptionsTab,
+  removeException,
+} from '../../../tasks/rule_details';
+import {
+  addExceptionEntryFieldValue,
+  addExceptionEntryFieldValueValue,
+  addExceptionFlyoutItemName,
+  submitNewExceptionItem,
+} from '../../../tasks/exceptions';
+import {
+  ADD_AND_BTN,
+  ADD_OR_BTN,
+  EXCEPTION_CARD_ITEM_NAME,
+  EXCEPTION_CARD_ITEM_CONDITIONS,
+  EXCEPTION_ITEM_VIEWER_CONTAINER,
+} from '../../../screens/exceptions';
+
+import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../../urls/navigation';
+
+describe(
+  'Add multiple conditions and validate the generated exceptions',
+  { testIsolation: false },
+  () => {
+    before(() => {
+      esArchiverResetKibana();
+      login();
+      deleteAlertsAndRules();
+      // At least create Rule with exceptions_list to be able to view created exceptions
+      createRule({
+        ...getNewRule(),
+        query: 'agent.name:*',
+        index: ['exceptions*'],
+        exceptions_list: [],
+        rule_id: '2',
+      });
+      visitWithoutDateRange(DETECTIONS_RULE_MANAGEMENT_URL);
+      goToRuleDetails();
+      goToExceptionsTab();
+    });
+
+    after(() => {
+      esArchiverUnload('exceptions');
+    });
+    const exceptionName = 'My item name';
+
+    it('Use multipe AND conditions and validate it generates one exception', () => {
+      // open add exception modal
+      openExceptionFlyoutFromEmptyViewerPrompt();
+
+      // add exception item name
+      addExceptionFlyoutItemName(exceptionName);
+
+      addExceptionEntryFieldValue('agent.name', 0);
+      addExceptionEntryFieldValueValue('foo', 0);
+
+      cy.get(ADD_AND_BTN).click();
+
+      addExceptionEntryFieldValue('@timestamp', 1);
+      addExceptionEntryFieldValueValue('123', 1);
+
+      submitNewExceptionItem();
+
+      // Only one Exception should generated
+      cy.get(EXCEPTION_ITEM_VIEWER_CONTAINER).should('have.length', 1);
+
+      // validate the And operator is displayed correctly
+      cy.get(EXCEPTION_CARD_ITEM_NAME).should('have.text', exceptionName);
+      cy.get(EXCEPTION_CARD_ITEM_CONDITIONS).should(
+        'have.text',
+        ' agent.nameIS fooAND @timestampIS 123'
+      );
+      removeException();
```

Comment:
Is it possible to move this into an after hook?

Reply:
Removed the action since we are using the ` esArchiverResetKibana`